webapp/src/js/actions/global.js

```diff
@@ -151,10 +159,40 @@ angular.module('inboxServices').factory('GlobalActions',
         });
       }
 
+      /**
+       * Unset the selected item without navigation
+       */
+      function unsetSelected() {
+        setShowContent(false);
+        setLoadingContent(false);
+        setShowActionBar(false);
+        setTitle();
+        dispatch({ type: actionTypes.CLEAR_SELECTED });
+        LiveList['contacts'].clearSelected();
+        LiveList['contact-search'].clearSelected();
+        LiveList['reports'].clearSelected();
+        LiveList['report-search'].clearSelected();
+        $('#reports-list input[type="checkbox"]').prop('checked', false);
+      }
+
+      /**
+       * Clear the selected item - may update the URL
+       */
+      function clearSelected() {
```

Comment:
@dianabarsan was actually talking about these function names at standup, I think they need clarification? Shouldn't `clearSelected` *always* call `unsetSelected` along with other things that it's doing?

Reply:
Yeah this is a mess. I've gone through and cleaned it up and renamed the `clearSelected` function to `navigateBack` which is actually what it's trying to do.